### ISSUE TITLE
Blank function name shown on "faas-cli list" whilst container is being removed

### ISSUE THREAD

**Issue by alexellis** [openfaas/faasd#58]
## Expected Behaviour

A function to show up whilst being deleted, or to be completely removed during deletion.

## Current Behaviour

A blank function name is shown in the place of the function being deleted on "faas-cli list" whilst the container is being removed (up to `write_timeout` seconds)

## Steps to Reproduce (for bugs)

1. `faas-cli store deploy figlet`
2. `faas-cli list`

```
alex@alex-nuc8:/tmp$ faas-cli list
Function                      	Invocations    	Replicas
figlet                        	0              	1   
```

3. `faas-cli rm figlet &`
4. `faas-cli list`

```
alex@alex-nuc8:/tmp$ faas-cli list
Function                      	Invocations    	Replicas
                              	0              	0    
```

**Comment by alexellis** [comment]
@carlosedp PTAL?

**Comment by carlosedp** [comment]
Looking at this.

**Comment by carlosedp** [comment]
Created PR https://github.com/openfaas/faasd/pull/61 to address this.

**Comment by carlosedp** [comment]
Derek close: Fixed by #61

**Comment by carlosedp** [comment]
Derek close: Fixed by #61
